stracer/src/main/scala/com/evolutiongaming/stracer/util/Probability.scala

```diff
@@ -0,0 +1,33 @@
+package com.evolutiongaming.stracer.util
+
+import com.evolutiongaming.stracer.Sampling
+import com.evolutiongaming.stracer.Sampling._
+
+trait Probability {
+  def apply(sampling: Option[Sampling]): Double
+}
+
+object Probability {
+
+  val defaultProbabilityValue = 0.01
+  val alwaysProbabilityValue = 1.0
+  val neverProbabilityValue = -1.0
```

Comment:
how this is supposed to work?

Reply:
we are checking that current probability is higher than generated double value. double value always generated in range [0.0, 1.0), so -1.0 will always be considered as 'not pass' for traces